internal/processor/filterconfig/config.go

```diff
@@ -71,7 +71,7 @@ type MatchProperties struct {
 	// Config configures the matching patterns used when matching span properties.
 	filterset.Config `mapstructure:",squash"`
 
-	// Note: For spans, one of Services, SpanNames or Attributes must be specified with a
+	// Note: For spans, one of Services, SpanNames, Attributes, Resources or Libraries must be specified with a
 	// non-empty value for a valid configuration.
 
 	// For logs, one of LogNames or Attributes must be specified with a
```

Comment:
Update comment.

Reply:
what exactly?

Reply:
```suggestion
	// For logs, one of LogNames, Attributes, Resources or Libraries must be specified with a
```
Don't we now support Resource and Libraries for logs too?